docs/section6/Rule6-12.md

```diff
@@ -0,0 +1,36 @@
+
+# Lighting - Rule 6-12
+
+**Rule ID:** 6-12  
+**Rule Description:** Proposed building is modeled with daylighting controls  
+**Appendix G Section:** Section 6 Lighting  
+**Appendix G Section Reference:** Section G3.1-6(h) Lighting: Modeling Requirements for the Proposed design  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+## Rule Logic: 
+
+- Check if each space has window or skylight in the building segment in the Proposed model: `For zone_p in P_RMR...zones:`
+
+  - Get surfaces in zone: `surface_p in zone_p.surfaces`
+
+    - Check if the surface is exterior and has openings: `if ( surface_p.adjacent_to == AMBIENT ) AND ( COUNT( surface_p.fenestration_subsurfaces ) >= 1 ): daylight_flag_p == TRUE`
+
+  - For each space in zone: `space_p in zone_p.spaces:`
+
+    - Get interior_lighting in space: `interior_lighting_p = space_p.interior_lighting`
+
+      - Check if any interior_lighting has daylight control: `if ( lighting.daylighting_control_type != "NONE" for lighting in interior_lighting_p ): has_daylight_control_flag == TRUE`
+
+        **Rule Assertion:** For each zone in the Proposed model:  
+
+        - Case 1, the zone has window or skylight and daylight control is modeled: `daylight_control_flag == TRUE AND has_daylight_control_flag == TRUE: PASS`
+
```

Comment:
I think this should be "CAUTION", because we are verifying whether ANY of the spaces included in the zone have daylighting controls, but then may ALL require daylighting controls. So we cannot confidently say whether the mandatory lighting control requirements are met.

Reply:
as discussed on RDS call, zones can have multiple spaces and if daylighting is specified in at least one the spaces and if atleast one the spaces has windows, then PASS. But this is not necessarily pass as we cannot verify that the space with daylighting controls is also the space with windows, hence CAUTION outcome.